accounts/abi/bind/base.go

```diff
@@ -340,6 +356,30 @@ func (c *BoundContract) UnpackLog(out interface{}, event string, log types.Log)
 	return parseTopics(out, indexed, log.Topics[1:])
 }
 
+// preparePrivateTransaction replace the payload of private transaction to the hash from Tessera/Constellation
+func (c *BoundContract) preparePrivateTransaction(tx *types.Transaction, payload []byte) *types.Transaction {
+	var privateTx *types.Transaction
+	if tx.To() == nil {
+		privateTx = types.NewContractCreation(tx.Nonce(), tx.Value(), tx.Gas(), tx.GasPrice(), payload)
+	} else {
+		privateTx = types.NewTransaction(tx.Nonce(), c.address, tx.Value(), tx.Gas(), tx.GasPrice(), payload)
+	}
+	privateTx.SetPrivate()
+	return privateTx
+}
+
+// sendPrivateTx send the private transaction to Tessera/Constellation
+func sendPrivateTx(tx *types.Transaction, privateFrom string, privateFor []string) ([]byte, error) {
+	if private.P == nil {
+		return nil, errors.New("transaction manager not set up")
+	}
+	data, err := private.P.Send(tx.Data(), privateFrom, privateFor)
```

Comment:
this calls `/sendraw` which is only meant for Quorum node.
Native DApps can only call `/storeraw`